app/uploaders/preneed_attachment_uploader.rb

```diff
@@ -36,7 +36,7 @@ def store_dir
   end
 
   def filename
-    "#{super.chomp(File.extname(super))}.pdf" if original_filename.present?
+    "#{super.chomp(File.extname(super))}.pdf"
```

Comment:
Thanks @mshibuya for the [quick fix](https://github.com/department-of-veterans-affairs/vets-api/pull/15118)